lms/services/h_api.py

```diff
@@ -77,7 +78,7 @@ def update_user(self, h_user):
             data={"display_name": h_user.display_name},
         )
```

Comment:
~I think in future it would be totally Ok for the HUser to know how to turn itself into an H compatible dict (`as_dict()`?) as all the parts come from it.~

Yeah ignore that. This is the H-API. While this is in charge of making API calls, we don't want to have to go hunting around for what the payload is in other files.

This might be more appropriate when the Bulk API is in place.

Reply:
namedtuples do come with an `_asdict()` method for free and the resulting dict happens to be close to (but not exactly) what's needed for the h create user API. But not what the update user API needs. It's probably fine for `HAPIService` to handle serializing things into the necessary formats for the different API calls though, as you say